### PR TITLE
Support Laravel 5.2 - closes #71

### LINKED ISSUE #71: Laravel 5.2
Add compatibility for Laravel 5.2

### PR DESCRIPTION
**Pull request by jbrooksuk**

### CODE REVIEW

File: composer.json [hunk 1]
```diff
@@ -9,12 +9,12 @@
     ],
     "require": {
         "php": ">=5.4.0",
-        "illuminate/support": "4.2.*|5.0.*|5.1.*"
+        "illuminate/support": "4.2.*|5.0.*|5.1.*|5.2.*"
```
Comment (arthurkirkosa) [inline composer.json]: Wouldn't this work just fine?
```
"illuminate/support": "^4.2.0"
```

Comment (jbrooksuk) [inline composer.json]: I don't think so, see the Composer documentation for [caret](https://getcomposer.org/doc/articles/versions.md#caret).

Comment (arthurkirkosa) [inline composer.json]: I see.

Maybe **>=4.2** or **4-5** ?

Comment (jbrooksuk) [inline composer.json]: Regardless, it'd be far cleaner and easier to read when it's explicitly stated.

Comment (arthurkirkosa) [inline composer.json]: True. But when 5.3 comes out you'll need to change this file.

http://semver.mwl.be/#?package=illuminate%2Fsupport&version=%3E%3D4.2&minimum-stability=stable

Comment (jbrooksuk) [inline composer.json]: And? What if 5.3 is backwards incompatible with something. You're assuming it won't break (it very likely will).

Comment (HipsterJazzbo) [inline composer.json]: Hey guys, FWIW I've moved this package to HipsterJazzbo/Landlord and made it 5.2 only.

Comment (arthurkirkosa) [inline composer.json]: Will this package get more updates? Or it's best to follow Landlord?

Comment (HipsterJazzbo) [inline composer.json]: All of this scope stuff changed from nightmare hellscape to pleasant meadow full of flowers in 5.2, so I'd recommend following Landlord. I'll leave this package here for legacy people but it won't get any updates, and it's marked as abandoned in packagist.

Comment (jbrooksuk) [inline composer.json]: :+1:

**Pull request merged**